package.json

```diff
@@ -24,7 +24,7 @@
         "asn1-parser": "1.1.8",
         "axios": "0.23.0",
         "buffer": "6.0.3",
-        "keccak": "3.0.1",
+        "keccak": "3.0.2",
```

Comment:
what's the reason to upgrade `keccak`?

Reply:
version 3.0.2 replaces `stream` with `readable-stream` so it doesn't require a polyfill